.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,55 @@
+name: Upgrade Requirements
+
+on:
+  schedule:
+    # will start the job at 01:30 UTC every Wednesday
+    - cron: "30 1 * * 3"
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Target branch to create requirements PR against"
+        required: true
+        default: 'main'
+
+jobs:
+  upgrade_requirements:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        python-version: ["3.8"]
+
+    steps:
+      - name: setup target branch
+        run: echo "target_branch=$(if ['${{ github.event.inputs.branch }}' = '']; then echo 'main'; else echo '${{ github.event.inputs.branch }}'; fi)" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v1
+        with:
+          ref: ${{ env.target_branch }}
+
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: make upgrade
+        run: |
+          cd $GITHUB_WORKSPACE
+          make upgrade
+
+      - name: setup testeng-ci
+        run: |
+          git clone https://github.com/edx/testeng-ci.git
```

Comment:
Is it correct for us to depend on this?
Should this library be on the openedx account?

Reply:
This repo, for some reason, still belongs to the edx organization, and there isn't a repo in the openedx organization. So should we raise an issue about this?

Reply:
Let's ask @openedx/tcril-engineering.

Is a dependency on this edx specific repo something expected when trying to set up the weekly update bot?

Reply:
2U's arch teams are still maintaining the underlying tooling that powers this so that is indeed the correct repo to clone for now.